fitlins/workflows/base.py

```diff
@@ -57,17 +57,16 @@ def init_fitlins_wf(bids_dir, derivatives, out_dir, analysis_level, space,
         name='getter')
 
     if smoothing:
-        smoothing_params = smoothing.split(':', 1)
-        if smoothing_params[0] != 'iso':
-            raise ValueError(f"Unknown smoothing type {smoothing_params[0]}")
-        smoothing_fwhm = float(smoothing_params[1])
+        smoothing_params = smoothing.split(':', 2)
+        if smoothing_params[1] != 'iso':
+            raise ValueError(f"Unknown smoothing type {smoothing_params[1]}")
+        smoothing_fwhm = float(smoothing_params[2])
+        smoothing_level = int(smoothing_params[0])
```

Comment:
Shouldn't be an `int`.

```suggestion
        smoothing_level = smoothing_params[0]
```